src/Test.UnitTests.Sarif.Driver/Sdk/ExportConfigurationCommandBaseTests.cs

```diff
@@ -62,12 +63,8 @@ public void ExportConfigurationCommandBase_VerifyDefaultConfiguration()
 
             configuration.Should().NotBeNull();
 
-            // We only have a single rule that provides options. Every rule
-            // provides a default setting, however, that controls whether 
-            // a rule is enabled or forced to a particular warning level.
-            // We therefore expect two top-level items in the configuration,
-            // each of which is a properties collection for a rule.
-            configuration.Keys.Count.Should().Be(3);
+            // We export two rules, FunctionlessTestRule and TestRule
+            configuration.Keys.Count.Should().Be(2);
```

Comment:
>2 [](start = 49, length = 1)

Rather than hard-coding the constant, and having to explain twice why it has this value, is there some way you can independently count the exports so you have
```
configuration.Keys.Count.Should().Be(NumberOfExportedRules());
```

Reply:
Death to magic numbers. Added a helper that simply counts types in the test assembly decorated with MEF ExportAttribute,, sufficient for now.

---
In reply to: [322893663](https://github.com/microsoft/sarif-sdk/pull/1670#discussion_r322893663) [](ancestors = 322893663)